src/main/scala/latis/server/CapabilitiesService.scala

```diff
@@ -10,6 +13,30 @@ class CapabilitiesService[F[_]: Effect] extends Http4sDsl[F] {
   val service: HttpService[F] =
     HttpService[F] {
       case GET -> Root / "capabilities" =>
-        Ok("Hello from HAPI!")
+        val capabilities =
+          Capabilities("2.0", Status.`1200`, List("binary", "csv", "json"))
```

Comment:
I think I'll factor out the version of the HAPI spec. I told myself I'd do that when I used the version in more than one place, but I already am!